tests/tests.dockerfile

```diff
@@ -1,15 +1,35 @@
 FROM node:8
 
-WORKDIR /workspace
-
-COPY package.json /workspace
-COPY tests/tests_docker.sh /workspace/tests_docker.sh
+ARG MONTAGU_GIT_ID="UNKNOWN"
+ARG MONTAGU_GIT_BRANCH="UNKNOWN"
 
-COPY ./resources /workspace/resources
-COPY ./tests /workspace/tests
+ENV MONTAGU_GIT_ID=$MONTAGU_GIT_ID
+ENV MONTAGU_GIT_BRANCH=$MONTAGU_GIT_BRANCH
 
 # This env var is needed for the custom reporter to log to teamcity
 ENV TEAMCITY_VERSION="teamcity"
 
-ENTRYPOINT ["./tests_docker.sh"]
+# Install docker
```

Comment:
Occurs to me that this node + docker image is exactly what we need in both webapps and here, so we should probably pull this first bit out into it's own repo as a base image that they both can inherit from, like we've done with https://github.com/vimc/openjdk-libsodium

Reply:
Ok, have added such a repo and used it as the base image
https://github.com/vimc/node-docker